### PR TITLE
[hal] Rename ClearColor::Type to ClearColor::Stype for consistency

### LINKED ISSUE #2781: Rename ClearColor::Int to ClearColor::Sint for consistency
breaking change...

### PR DESCRIPTION
**Pull request by BourgondAries**
Fixes #2781
PR checklist:
- [x] `make` succeeds (on *nix)
- [x] `make reftests` succeeds
- [x] tested examples with the following backends: vulkan
- [x] `rustfmt` run on changed code

### CODE REVIEW

File: src/backend/dx11/src/lib.rs [hunk 1]
```diff
@@ -1484,11 +1484,11 @@ impl hal::command::RawCommandBuffer<Backend> for CommandBuffer {
                     | ChannelType::Sfloat
                     | ChannelType::Uscaled
                     | ChannelType::Sscaled
-                    | ChannelType::Srgb => ClearColor::Float(unsafe { raw_clear.float32 }),
+                    | ChannelType::Srgb => ClearColor::Sfloat(unsafe { raw_clear.float32 }),
 
-                    ChannelType::Uint => ClearColor::Uint(unsafe { raw_clear.uint32 }),
+                    ChannelType::Uint => ClearColor::Suint(unsafe { raw_clear.uint32 }),
```
Comment (kvark) [inline src/backend/dx11/src/lib.rs]: I don't think "Suint" makes sense :) Should be "Uint"

Comment (BourgondAries) [reply src/backend/dx11/src/lib.rs]: So I guess the `S` stands for signed? I was under the impression it meant `Single` like Sfloat, Dfloat for double.

**Comment by kvark** [comment]
Right, S is for Signed

**Comment by BourgondAries** [comment]
Can this be merged now @kvark?

**Comment by BourgondAries** [comment]
Should be good now

**Pull request merged**